yr/tutorials/Number_Guessing_Game.md

```diff
@@ -0,0 +1,420 @@
+
+<hint markdown="block" title="What is an Alexa Skill?">
+
+An Alexa Skill is a <strong>voice-based application</strong> for Amazon Alexa-enabled devices, such as Amazon Echo Dots for Alexa-enabled smartphones. Don't worry about developing and testing Alexa Skills if you don't have your own Alexa-enabled devices. We'll show you how to use the Amazon  Developer page to test your skills using a virtual Alexa that can be run on any dvice!
+</hint>
+
+<p style="padding-bottom: 7px"></p>
+
+# Setup
+
+## Create an Amazon Developer Account
+If you haven't already, follow the instructions [here](https://drive.google.com/file/d/1OINgdsvgb42Or9AoZ3bJDLO5tQ6I6yTB/view) to create a <u>free</u> <strong>Amazon Developer Account</strong>. 
+This will allow you to save any Alexa Skills to your account and use your skills on any Alexa-enabled devices that you have linked to your Amazon account.
+<p style="padding-bottom: 7px"></p>
+
+## Create a New Project
+Navigate to (CHANGE SERVER LINK). 
+If you hav another project open, go to <strong>My Projects</strong> menu and choose <strong>New Project</strong>
+
+Insert picture here
+
+In the resulting popup, enter our project name, <strong>number_guesser</strong>. Then, click OK.
+
+Insert picture here
+<p style="padding-bottom: 7px"></p>
+
+# Creating Skill and First Custom Intent: *quizmeIntent*
+
+## Add a Skill
+To start, we want to add a new Skill to our project. Select the <strong>Add...</strong> dropdown next to Screen 1, and select <strong>Skill</strong>.
+
+Insert Picture Here
+
+In the resulting popup, enter our Skill's <strong>Invocation name</strong>, "guessing game host". Then, select <strong>OK</strong>. 
+
+Insert Picture Here.
+
+The first thing we want to teach our skill to listen out for is its <strong>Invocation Name</strong>. 
+For this demonstration, we named our skill "guessing game host". This means when we want to call our skill, we will say: 
+
+<center> “Alexa, tell <span style="background-color: yellow"> guessing game host</span> to <span style="background-color: LightPink"> do something</span>” </center>
+<p style="padding-bottom: 7px"></p>
+
+<hint markdown="block" title="What is an Invocation Name?">
+Just like how every mobile app needs to have a name, so does our custom Alexa Skill. An Invocation Name is just the name of the Skill that we are making, and is used to “invoke” our skill. The structure of any command you will tell Alexa is:
+
+<center> “Alexa, tell <span style="background-color: yellow"> (Invocation Name)</span> to <span style="background-color: LightPink"> ...</span>” </center>
+
+The invocation name is what will help Alexa tell which Skill it needs to use, so make sure that every skill you make has a unique name.
```

Comment:
Kind of in the same line with what I mention before, but if the tutorials have a gradual step, then students already know what invocation name is at this point.